JobBot-server/jobbot/jobbot/src/test/java/org/dizzybot/jobbot/entities/AccountTest.java

```diff
@@ -0,0 +1,52 @@
+package org.dizzybot.jobbot.entities;
+
+import jakarta.validation.ConstraintViolation;
+import jakarta.validation.Validation;
+import jakarta.validation.Validator;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.List;
+import java.util.Set;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class AccountTest {
+
+    private User user;
+
+    private Account account;
+
+    private Validator validator = Validation.buildDefaultValidatorFactory().getValidator();
+
+    @Before
+    public void beforeEach() {
+        this.user = new User("Dizzy Bot", "Password", "dizzybot@email.com");
+        this.account = this.user.account;
```

Comment:
`this.user.getAccount()`, do not directly access attribute, access attribute using getter.